### DISCUSSION TITLE
라이트 세팅 관련

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#207] [5. Q&A]
안녕하세요.
제페토 월드 환경 구성 중에 라이트 설정 관련하여 문의드립니다.

1. 제페토 월드에서 유니티 라이트맵 사용이 가능한가요?
2. 제페토 월드에서 사용 가능한 라이트 개수의 제한이 있나요?

확인 부탁드립니다.
감사합니다.

**Comment by moondory77-naverz** [comment]
안녕하세요. 
현재 제페토 월드 개발에서, 라이트 맵 사용이 가능하며 특별한 제한은 없습니다.
감사합니다.

**Reply by strangestraw** [reply]
감사합니다!
